Makefile

```diff
@@ -193,7 +180,10 @@ check-eds: fmt vet lint
 	go build -ldflags '${LDFLAGS}' -o bin/check-eds ./cmd/check-eds/main.go
 
 bin/kubebuilder:
-	./hack/install-kubebuilder.sh 2.3.2
+	./hack/install-kubebuilder.sh 2.3.2 ./bin
+
+bin/kubebuilder-tools:
```

Comment:
the `kubebuilder` version that we were using previously did provide ARM support. But the version that do support ARM, does release the testing binary (api-server, etcd...) in a dedicated new package: kubebuild-tools